tests/IBFE/instrument_panel_01_3d.output

```diff
@@ -0,0 +1,51 @@
+Number of elements: 64
+
+IBFEMethod: mesh part 0 is using FIRST order LAGRANGE finite elements.
+
+IBHierarchyIntegrator::initializePatchHierarchy(): tag_buffer = 3
+INSStaggeredHierarchyIntegrator::initializeCompositeHierarchyData():
+  projecting the interpolated velocity field
+INSStaggeredHierarchyIntegrator::regridProjection(): regrid projection solve residual norm        = 2.58993e-05
+
+
+Writing visualization files...
+
+test 0
+Warning in file ``/Users/boyceg/code/IBAMR/lib/../src/IB/IBFEInstrumentPanel.cpp'' at line 658
```

Comment:
Mind if I push a change to suppress this warning? This path isn't going to work on other people's machines :)

Reply:
Please do

Reply:
Whoops --- it doesn't even work on my own machine!

Reply:
I'm working on it now - we have a class `TestAppender` in `tests/tests.h` which works around exactly this issue with SAMRAI.

Reply:
I fixed the warnings and a few other minor things.